### PR TITLE
Apply column filtering to v12

### LINKED ISSUE #597: Apply FGA (column filtering) to v12

### PR DESCRIPTION
**Pull request by laurent-thiebaud-gisaia**
Merging branch.
Fix #597 and #586

### CODE REVIEW

**Comment by laurent-thiebaud-gisaia** [comment]
Developments have been applied from branch 11.6.x .
Still multi-collections to be managed.

File: arlas-admin/src/main/java/io/arlas/server/task/CollectionAutoDiscover.java [hunk 1]
```diff
@@ -61,7 +62,7 @@ public void execute(ImmutableMultimap<String, String> arg0, PrintWriter arg1) th
             List<CollectionReferenceDescription> discoveredCollections = admin.getAllIndecesAsCollections();
```
Comment (alainbodiguel) [inline arlas-admin/src/main/java/io/arlas/server/task/CollectionAutoDiscover.java]: Should we not correct Indeces -> Indices ?

Comment (laurent-thiebaud-gisaia) [reply arlas-admin/src/main/java/io/arlas/server/task/CollectionAutoDiscover.java]: For a reason I don't understand, there are 16 repetitions of "indeces" in the code, I think it should rather be part of a dedicated PR

Comment (alainbodiguel) [reply arlas-admin/src/main/java/io/arlas/server/task/CollectionAutoDiscover.java]: ok for me!

File: arlas-opensearch/src/main/java/io/arlas/server/rest/explore/opensearch/OpenSearchDescriptorService.java [hunk 2]
```diff
@@ -128,7 +134,7 @@ public Response opensearch(
             description.syndicationRight = os.syndicationRight;
             description.tags = os.tags;
         }
-        addURLs(prefix, description.url, new ElasticAdmin(exploreServices.getClient()).describeCollection(cr).properties, new Stack<>());
+        addURLs(prefix, description.url, admin.describeCollection(cr, columnFilter).properties, new Stack<>());
```
Comment (MohamedHamouGisaia) [inline arlas-opensearch/src/main/java/io/arlas/server/rest/explore/opensearch/OpenSearchDescriptorService.java]: We can use `exploreServices.getElasticAdmin()`
we avoid creating another instance of the admini and the creation of `admin` variable

**Comment by MohamedHamouGisaia** [comment]
Documentation doesn't include the explanation of column-filter and collections

**Comment by laurent-thiebaud-gisaia** [comment]
_rebased to squash few commits_

**Pull request merged**